pkg/tfvars/azure/azure.go

```diff
@@ -76,7 +76,7 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 	}
 
 	controlPlaneUltraSSDEnabled := false
-	if masterConfig.UltraSSDCapability == machineapi.AzureUltraSSDCapabilityTrue {
+	if masterConfig.UltraSSDCapability == machineapi.AzureUltraSSDCapabilityEnabled {
 		controlPlaneUltraSSDEnabled = true
 	}
```

Comment:
We can refactor this code block out to where the assignment in the config becomes:

```go
		ControlPlaneUltraSSDEnabled:     masterConfig.UltraSSDCapability == machineapi.AzureUltraSSDCapabilityEnabled,

```

Reply:
That is not possible because the OpenShift API deviates from both Terraform and the Azure API precisely in this regard. Instead of "Enabled" and "Disabled", Terraform uses "True" and "False". As such, it is necessary to translate between the two. I deliberately did so here, above the assignment below, because the in-line syntax was long and not self evident.

Reply:
We could have done that, no doubt. All I'm doing in this PR is updating the API and making our code compatible.

Reply:
Ok. I will open a PR to refactor.